config/config.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+return [
+    'take' => env('TAKE', 12),
+    'pagination' => env('PAGINATION', 8),
+    'default_prioritize' =>  env('DEFAULT_PRIORITIZE', 1),
+    'default_server' => env('DEFAULT_SERVER', 1),
```

Comment:
```suggestion
    'take' => 12, take là cái gì đấy?
    'pagination' => 8,
```
ko cần đưa vào env nhé, em set config thôi